src/applications/post-911-gib-status/reducers/index.js

```diff
@@ -32,7 +32,7 @@ function post911GIBStatus(state = initialState, action) {
     case NO_CHAPTER33_RECORD_AVAILABLE:
       return set('availability', 'noChapter33Record', state);
     case GET_ENROLLMENT_DATA_FAILURE:
-      return set('availability', 'unavailable', state);
+      return set('availability', 'getEnrollmentDataFailure', state);
```

Comment:
it felt counterintuitive for this to be the only state name that didn't match the action constant, so switch it to conform